.github/workflows/build-docs.yml

```diff
@@ -0,0 +1,31 @@
+name: build-docs
+on:
+  pull_request:
+    paths:
+      - .github/workflows/**
+      - docs/**
```

Comment:
we didn't catch the error in the PR it was introduced (#5517) because this job was only been triggered if a modification to the modin's code itself is made:
https://github.com/modin-project/modin/blob/ff53445e1baa916d4b4ac6f2ecab21b86f5ccf02/.github/workflows/ci.yml#L7-L17

Now I've added this trigger to run this job if `docs/**` were modified.